deblur/test/test_workflow.py

```diff
@@ -196,7 +188,7 @@ def test_remove_artifacts_seqs(self):
                         "TCGGCTTTGTAAATCCCTGGGTAAATAGGGT"),
                ("ref6", "TACCCGCAGCTCAAGTGGTGGTCGCTATTATTGAGCCTAAAACGTCCGTAG"
                         "TCGGCTTTGTAAATCCCTGGGTAAATCGGGT")]
-        ref_fp = join(self.working_dir, "ref.fasta")
+        ref_fp = join(self.working_dir, "ref2.fasta")
```

Comment:
just to enable debugging (so we know which file is from which stage...)

On Thu, Mar 17, 2016 at 4:45 AM, Evguenia Kopylova <notifications@github.com
> wrote:

> In deblur/test/test_workflow.py
> <https://github.com/biocore/deblur/pull/32#discussion_r56491246>:
>
> > @@ -196,7 +188,7 @@ def test_remove_artifacts_seqs(self):
> >                          "TCGGCTTTGTAAATCCCTGGGTAAATAGGGT"),
> >                 ("ref6", "TACCCGCAGCTCAAGTGGTGGTCGCTATTATTGAGCCTAAAACGTCCGTAG"
> >                          "TCGGCTTTGTAAATCCCTGGGTAAATCGGGT")]
> > -        ref_fp = join(self.working_dir, "ref.fasta")
> > +        ref_fp = join(self.working_dir, "ref2.fasta")
>
> what's the purpose of ref2, ref3, etc. over ref?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/deblur/pull/32/files/069a0cd16c523f104f10937cd3dd5fd5715c57bb#r56491246>
>